aws_advanced_python_wrapper/aws_secrets_manager_plugin.py

```diff
@@ -94,7 +98,13 @@ def force_connect(
         return self._connect(props, force_connect_func)
 
     def _connect(self, props: Properties, connect_func: Callable) -> Connection:
-        secret_fetched: bool = self._update_secret()
+        token_expiration_sec: int = WrapperProperties.SECRETS_MANAGER_EXPIRATION.get_int(props)
+        # if value is less than 0, default to one year
+        if token_expiration_sec < 0:
+            token_expiration_sec = AwsSecretsManagerPlugin._ONE_YEAR_IN_SECONDS
+        token_expiration_ns = token_expiration_sec * 1000
```

Comment:
Should this be ms instead of ns?

Reply:
that makes sense. Not sure why i thought there was 1000 ns in a second